src/transformers/visitors/toOrchestrationVisitor.ts

```diff
@@ -916,7 +1245,87 @@ const visitor = {
 
   FunctionCall: {
     enter(path: NodePath, state: any) {
-      const { node, parent } = path;
+      const { node, parent, scope } = path;
+      if(path.isInternalFunctionCall()) {
```

Comment:
Indentation and smushing (technical term :) ) below where possible would be good please